php/ice/public/index.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$di = new \Ice\Di();
+
+$di->loader
+    ->addNamespace('App', __DIR__ . '/../App')
+    ->register();
+
+$di->dispatcher
+    ->setNamespace('App');
+
+$di->router
+    ->setRoutes([
```

Comment:
@mruz a route is missing : `['GET', '/']`

Reply:
https://github.com/the-benchmarker/web-frameworks/blob/684ac4a6088f640d11820dda79b5d1bfa1cc3c21/php/ice/public/index.php#L14
It's defined in the optional action `[/{action}]`, so not needed to define it again. Default action `index` will be used.

Reply:
Strange then I've a server error on while requesting this endpoint

Reply:
Could you rebase and push to trigger`CI` ?

Reply:
Strange, travis do not start

Reply:
![Screenshot_20200419_183556](https://user-images.githubusercontent.com/2629882/79693723-a97f6a00-826c-11ea-83d4-9ce72d28bf3c.png)
I'm not sure why, but it says "Abuse detected" for all my commits https://github.com/travis-ci/travis-ci/issues/8626

Reply:
I've seen that, I sent an email to travis' support. I'll probably create a `PR` in this repository, and include your commits :stuck_out_tongue:

Reply:
this abuse was detected on my side, I donot know more, yet